### PR TITLE
fix vs2019 release builds

### PR DESCRIPTION
**Pull request by qndel**
good job microsoft!

### CODE REVIEW

**Comment by AJenbo** [comment]
Nice, but is it needed for all the variables?

**Comment by qndel** [comment]
no but it won't be a nice 1liner otherwise :P

File: Source/drlg_l3.cpp [hunk 1]
```diff
@@ -1970,6 +1970,9 @@ void FenceDoorFix()
 
 static void DRLG_L3Wood()
 {
+#if (_MSC_VER >= 1920) 
+	volatile // visual studio 2019 throws internal compiler error without it
```
Comment (AJenbo) [inline Source/drlg_l3.cpp:1974]:
```suggestion
	volatile // visual studio 2019 throws internal compiler error without it, see #708

```

**Comment by qndel** [comment]
```cpp
#if (_MSC_VER >= 1920) 
	volatile int j; // visual studio 2019 throws internal compiler error without it
#else
	int j;
#endif
	int i, x, y, xx, yy, rt, rp, x1, y1, x2, y2;
	```

**Comment by AJenbo** [comment]
Could also do:
```cpp
#if (_MSC_VER >= 1920) 
	volatile // visual studio 2019 throws internal compiler error without it, see #708
#endif
	int j;
	int i, x, y, xx, yy, rt, rp, x1, y1, x2, y2;
```
But on second thought let's not rearrange the code just to cater to a buggy compiler.

**Pull request merged**